aidants_connect_web/models.py

```diff
@@ -203,6 +203,41 @@ def normalize_birthplace(self):
         return self.birthplace
 
 
+class AutorisationDureeKeywords(models.TextChoices):
```

Comment:
je ne connaissais pas cette façon de faire. c'est des ENUM, un peu comme [ici](https://adamj.eu/tech/2020/01/27/moving-to-django-3-field-choices-enumeration-types/) ?

Reply:
Voui, je voulais tester :)

Reply:
d'acc ! et si on voulait utiliser ces valeurs dans un form, on peut y faire appel ? quid de le mettre dans les settings ? (je dis ça car je suis le nez dans `MandatDureeKeywords` qui aussi utilisé/dupliqué dans `forms.py`)

Reply:
L'approche la plus canonique est de déclarer ces choses "près du modèle", voire _dans la classe_ du modèle, histoire de n'avoir qu'à importer celui-ci pour qu'elles "viennent avec".

Alternativement, quand un projet commence à grossir, on voit souvent apparaître des choses comme un `constants.py` par exemple, qui centralisent toutes ces déclarations, et dont on peut les importer où qu'on le souhaite :)

Le fait de _dupliquer_ ça est clairement un anti-pattern par contre, à proscrire autant que possible ^^